docs/streams/developer-guide.html

```diff
@@ -1828,8 +1830,10 @@
             () -> 0, /* initializer */
             (aggKey, newValue, aggValue) -> aggValue + newValue, /* adder */
             (aggKey, oldValue, aggValue) -> aggValue - oldValue, /* subtractor */
-            Serdes.Integer(), /* serde for aggregate value */
-            "aggregated-table-store" /* state store name */);
+            Materialized.&lt;String, Integer, KeyValueStore&ltBytes, byte[]&gt;&gt;as("aggregated-table-store")
+                .withKeySerde(Serdes.String() /* serde for aggregate key */)
```

Comment:
why adding the key Serde? Was not there before.

Reply:
Typo:
```
KeyValueStore&ltBytes
```
This should be:
```
KeyValueStore&lt;Bytes
```

Reply:
@mjsax, unless the default serdes are set in the config and the aggregation uses the same Serde, then we need to specify both serdes